src/utils/rowTemplateColumns.ts

```diff
@@ -0,0 +1,359 @@
+        sortFunction(resultA, resultB) {
+          return (
+            Math.abs(resultA.cliffs_delta) - Math.abs(resultB.cliffs_delta)
+          );
+        },
+        tooltip:
+          'Cliff’s Delta quantifies the magnitude of the difference between Base and New values.',
+      },
+      {
+        name: 'Significance',
+        key: 'significance',
+        filter: true,
+        gridWidth: tableMannWhitneyConfigTestSubtestDiff.significanceGridWidth,
+        tooltip:
+          'Mann Whitney U test p-value indicating statistical significance. Mann Whitney U p-value < .05 indicates a statistically significant difference between Base and New.',
+        possibleValues: [
+          { label: 'Significant', key: 'significant' },
+          { label: 'Not Significant', key: 'not significant' },
+        ],
+        matchesFunction(result, valueKey) {
+          switch (valueKey) {
+            case 'none':
+              return !result.mann_whitney_test?.interpretation;
+            default: {
+              const label = this.possibleValues.find(
+                ({ key }) => key === valueKey,
+              )?.label;
+              return result.mann_whitney_test?.interpretation === label;
+            }
+          }
+        },
+        sortFunction(
+          resultA: MannWhitneyResultsItem,
+          resultB: MannWhitneyResultsItem,
+        ) {
+          if (
+            !resultA.mann_whitney_test?.pvalue ||
+            !resultB.mann_whitney_test?.pvalue
+          ) {
+            return 0;
+          } else {
+            return (
+              Math.abs(resultA.mann_whitney_test?.pvalue ?? 0) -
+              Math.abs(resultB.mann_whitney_test?.pvalue ?? 0)
+            );
+          }
+        },
+      },
+      {
+        name: 'Effect Size (%)',
+        key: 'effects',
+        gridWidth: '1.25fr',
+        sortFunction(
+          resultA: MannWhitneyResultsItem,
+          resultB: MannWhitneyResultsItem,
+        ) {
+          if (!resultA.cles?.cles || !resultB.cles?.cles) {
+            return 0;
+          } else {
+            return (
+              Math.abs(resultA.cles?.cles) - Math.abs(resultB.cles?.cles ?? 0)
+            );
+          }
+        },
+        tooltip:
+          'Common Language Effect Size (CLES) percentage is a measure of effect size. CLES >= 0.5 indicates probability Base > New.',
+      },
+      {
+        name: 'Total Runs',
+        key: 'runs',
+        gridWidth: '1fr',
+        tooltip: 'The total number of tasks/jobs that ran for this metric.',
+      },
+      // We use the real pixel value for the buttons, so that everything is improvement aligned.
+      {
+        key: 'buttons',
+        gridWidth: `calc(${tableMannWhitneyConfigTestSubtestDiff.colWidthMultiply} * 34px)`,
+      }, // 2 or 3 buttons, so at least 3*34px, but give more so that it can "breathe"
+      { key: 'expand', gridWidth: '34px' }, // 1 button
+    ];
+
+  switch (true) {
```

Comment:
This could just be an `if` statement like this:
```
if (testVersion === MANN_WHITNEY_U) {
    return columnsMannWhitneyConfiguration;
}
return columnsConfiguration;
```

It could be improved a bit further by making it by making it dynamically selected with an object keyed with test type (e.g. MANN_WHITNEY_U/STUDENT_T). But that's a big nit - no need to do that here, and there might be a better way than that to go about this (I'd have to look around).